libraries/chain/include/eosio/chain/webassembly/eos-vm-oc/eos-vm-oc.hpp

```diff
@@ -44,7 +44,7 @@ enum eosvmoc_exitcode : int {
    EOSVMOC_EXIT_EXCEPTION
 };
 
-static constexpr uint8_t current_codegen_version = 1;
+static constexpr uint8_t current_codegen_version = 2;
```

Comment:
This small change will effectively purge a pre-1.1 cache since on load,
https://github.com/AntelopeIO/spring/blob/39844283ed44e7d8764ac5ba25bda4cc45d95584/libraries/chain/webassembly/runtimes/eos-vm-oc/code_cache.cpp#L345-L354